hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -642,6 +642,15 @@
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
+                                <xs:attribute name="hot-restart-enabled" use="optional" type="xs:string">
+                                    <xs:annotation>
+                                        <xs:documentation>
+                                            This boolean parameter enables hot-restart feature when set as
+                                            `true`.
```

Comment:
No point in using backticks here, is there? Or is this scraped by some markdown-honoring tool?